src/packages.jl

```diff
@@ -58,6 +56,14 @@ function init_packagemanager()
       # put the new method in the first position
       meths = Globals.Download_Methods
       Wrappers.Add(meths, GapObj(r, recursive=true), 1)
+
+      # monkey patch PackageManager so that we can disable removal of
+      # package directories for debugging purposes
+      orig_PKGMAN_RemoveDir = Globals.PKGMAN_RemoveDir
+      replace_global!(:PKGMAN_RemoveDir, function(dir)
+        Globals.ValueOption(GapObj("debug")) == true && return
+        orig_PKGMAN_RemoveDir(dir)
```

Comment:
Of course we can remove this hack once we are using a PackageManager release with https://github.com/gap-packages/PackageManager/pull/119 or something equivalent in it